arch/arm64/boot/dts/qcom/msm8953-xiaomi-markw.dts

```diff
@@ -171,6 +178,26 @@
 	compatible = "xiaomi,markw-panel";
 };
 
+&sound_card {
+	model = "xiaomi-markw";
+
+	aux-devs = <&speaker_amp>;
+	audio-routing =
+		"AMIC1", "MIC BIAS External1",
+		"AMIC2", "MIC BIAS Internal2",
+		"AMIC3", "MIC BIAS External1";
+
+	pinctrl-names = "default", "sleep";
+	pinctrl-0 = <&cdc_pdm_lines_act &cdc_pdm_lines_2_act &cdc_pdm_comp_lines_act>;
+	pinctrl-1 = <&cdc_pdm_lines_sus &cdc_pdm_lines_2_sus &cdc_pdm_comp_lines_sus>;
```

Comment:
pinctrl-names below pinctrl-0 and pinctrl-1

Reply:
but it is above them in all other nodes

Reply:
Then the other nodes are in a wrong style too. Please fix them in the whole file, since it's also not upstream yet it should be fixed when the file gets upstreamed